Sources/MongoSwift/BSON/Document+Sequence.swift

```diff
@@ -280,11 +280,11 @@ public class DocumentIterator: IteratorProtocol {
         try newValue.writeToCurrentPosition(of: self)
     }
 
-    private static let BsonTypeMap: [BsonType: BsonValue.Type] = [
+    private static let BSONTypeMap: [BSONType: BSONValue.Type] = [
```

Comment:
this should probably be lowercase (my bad for the original naming.)
```suggestion
    private static let bsonTypeMap: [BSONType: BSONValue.Type] = [
```